--- config.mk.orig	Thu Oct 28 00:26:02 2021
+++ config.mk	Thu Oct 28 00:26:16 2021
@@ -14,9 +14,9 @@ XINERAMAFLAGS = -DXINERAMA
 
 # freetype
 FREETYPELIBS = -lfontconfig -lXft
-FREETYPEINC = /usr/include/freetype2
+#FREETYPEINC = /usr/include/freetype2
 # OpenBSD (uncomment)
-#FREETYPEINC = $(X11INC)/freetype2
+FREETYPEINC = $(X11INC)/freetype2
 
 # includes and libs
 INCS = -I$(X11INC) -I$(FREETYPEINC)
